test_dj_database_url.py

```diff
@@ -16,7 +18,10 @@ def test_postgres_parsing(self):
         url = 'postgres://uf07k1i6d8ia0v:wegauwhgeuioweg@ec2-107-21-253-135.compute-1.amazonaws.com:5431/d8r82722r2kuvn'
         url = dj_database_url.parse(url)
 
-        assert url['ENGINE'] == 'django.db.backends.postgresql_psycopg2'
+        if DJANGO_VERSION < (2, 0):
```

Comment:
This seems to be repeated everywhere. What if we have a constant at the module level that looks like:

```py
EXPECTED_ENGINE = 'django.db.backends.postgresql'
if DJANGO_VERSION < (2, 0):
    EXPECTED_ENGINE = 'django.db.backends.postgresql_pyscopg2'
```

And then we use `EXPECTED_ENGINE` in our assertions